.github/workflows/hive-tests.yml

```diff
@@ -16,6 +16,9 @@ jobs:
   hive-tests:
     runs-on: ubuntu-latest
     timeout-minutes: 60
+    concurrency:
```

Comment:
Any reason why this concurrency is not added for the other job defined in this file?

Reply:
Not a good reason. I think it was that I didn't really understand what the other tests were doing, and this is mostly a demo. Also, the other tests have a 20m timeout instead of 60m, so they're already perforce cheaper.

Happy to add them if you like although each of them will need to have a different name so that they don't trip on each other, so,
```
      group: ${{ github.workflow }}-${{ github.job }}-${{ github.event.pull_request.number }}
```

would probably be better

Reply:
For consistency, I've applied this everywhere. -- Otherwise there's a risk of someone doing a copy-paste and having one job shoot the other.

Reply:
Oops. `${{ github.job }}` isn't available... So i'm just going to be explicit to make sure that nothing trips on anything.